modules/serverless-ossm-v2x-jwt.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * serverless/networking/serverless-ossm-jwt.adoc
+
+[id="serverless-ossm-v2x-jwt_{context}"]
+= Using JSON Web Token authentication with {ProductShortName} 2.x and {ServerlessProductName}
+
+.Procedure
+
+. Create the `RequestAuthentication` resource by copying the following code into a YAML file:
++
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: RequestAuthentication
+metadata:
+  name: jwt-example
+spec:
+  jwtRules:
+  - issuer: testing@secure.istio.io
+    jwksUri: https://raw.githubusercontent.com/istio/istio/release-1.8/security/tools/jwt/samples/jwks.json
+----
+. Apply the `RequestAuthentication` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+. To allow access to the `RequestAuthenticaton` resource from system pods, copy the following `AuthorizationPolicy` resources into a YAML file:
++
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allowlist-by-paths
+spec:
+  action: ALLOW
+  rules:
+  - to:
+    - operation:
+        paths:
+        - /metrics   # The path to collect metrics by system pod.
+        - /healthz   # The path to probe by system pod.
+----
+. Apply the `AuthorizationPolicy` resource YAML file:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+. Copy the following `AuthorizationPolicy` resources into a YAML file:
++
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: require-jwt
+spec:
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+       requestPrincipals: ["testing@secure.istio.io/testing@secure.istio.io"]
+----
+. Apply the `AuthorizationPolicy` resource YAML file:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+.Verification steps
+
+. If you try to use a `curl` request to get the Knative service URL, it is denied.
++
+[source,terminal]
+----
+$ curl http://hello-example-default.apps.mycluster.example.com/
```

Comment:
Let's use the same example service from `include::modules/serverless-service-mesh-enable-sidecar-injection.adoc`:

```suggestion
$ curl http://hello-example-1-default.apps.mycluster.example.com/
```